ddcz/templates/postal_service/components/new_letter.html

```diff
@@ -0,0 +1,17 @@
+<section class="postal_service_send">
+    <form method="post">
+    {% csrf_token %}
+    <input type="hidden" name="fid" value="{{ send_id }}">
+    
+    <label for="whom">Drakovi stačí znát příjemce:</label>
+    <select name="whom">
+        {% for post_user in users %}
```

Comment:
This will not work: 

1) You are loading all users into memory, which will not fit
2) You are creating a list of all users on DDCZ. I am quite sure a drop-down with 10k elements will freeze some browsers, especially on the mobile.